bundles-lib/src/test/java/org/apache/metron/bundles/integration/components/MRComponent.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.metron.bundles.integration.components;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hdfs.MiniDFSCluster;
+import org.apache.hadoop.yarn.conf.YarnConfiguration;
+import org.apache.metron.bundles.integration.InMemoryComponent;
+
+import java.io.IOException;
+
+public class MRComponent implements InMemoryComponent {
```

Comment:
So, as mentioned above - the idea at the start was to create something that make end up being a new shared library between NiFi and Metron down the road.  Now, I didn't hold my own feet to the fire on that, but there are some areas where that thinking comes through.  This is one.  I did not want to create dependencies between bundle-lib and any metron libs.  So, where necessary I created copies.  I know this isn't great, and the original goal may just be fantasy, but that is why.